src/test/java/Homework18.java

```diff
@@ -0,0 +1,38 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest {
+    @Test
+
+    public void addSongToPlaylist() throws InterruptedException {
+        String newSongAddedToPlaylistNotification = "Added 1 song into";
+        //Opens url WebPage
+        accessUrlPage();
+        //Enters email credentials into webpage Email space
+        provideEmailCredentials("james.patterson@testpro.io");
+        providePasswordCredentials("te$t$tudent");
+
+        loginButton();
+
+//        songSearch("Pluto");
+//
+//        viewAllSongs();
+//
+//        firstSongSelection();
+//
+//        addToButton();
+//
+//        selectedSongAddedToPlaylist();
+//
+//        Assert.assertTrue(addedSongMessage().contains(newSongAddedToPlaylistNotification));
+//
+//        homeButton();
+
+//        playSong();
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.